### PR TITLE
Added a Ton Macron form workflow and action

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: app/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -96,3 +96,20 @@ newsletter.invitation.form.last_name: Votre nom
 newsletter.invitation.form.guest: E-mail d'un ami
 newsletter.invitation.form.invite: J'invite des amis
 newsletter.invitation.form.invite_sent: Votre invitation a bien été envoyée ! | Vos %count% invitations ont bien été envoyées !
+
+#
+# Ton Macron
+#
+ton_macron.invitation_form.friend_first_name: Son prénom
+ton_macron.invitation_form.friend_age: Son âge
+ton_macron.invitation_form.friend_position: Son status professionnel
```
Comment (hhamon) [inline app/Resources/translations/messages.fr.yml]: `Statut`

File: app/config/config.yml [hunk 2]
```diff
@@ -3,13 +3,7 @@ imports:
...
     - { resource: adherent.yml }
     - { resource: assets_version.yml }
-    - { resource: services/admin.xml }
-    - { resource: services/cms.xml }
-    - { resource: services/community.xml }
-    - { resource: services/external_services.xml }
-    - { resource: services/platform.xml }
-    - { resource: services/repositories.xml }
-    - { resource: services/system.xml }
+    - { resource: services/ }
```
Comment (hhamon) [inline app/config/config.yml]: Don't you have to specify `type: directory` or something similar?

File: src/AppBundle/Form/TonMacronInvitationType.php [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class TonMacronInvitationType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        switch ($options['transition']) {
+            case InvitationProcessor::TRANSITION_FILL_INFO:
```
Comment (hhamon) [inline src/AppBundle/Form/TonMacronInvitationType.php]: can you make separate private method please:

```
case XXX:
    $this->configureUserInformatioForm($builder, $options);
case YYY:
    $this->configureSomethingElseForm($builder, $options);
```

File: app/Resources/translations/messages.fr.yml (hunk 1 above)
Comment (HeahDude) [inline app/Resources/translations/messages.fr.yml]: Fixed, thanks.

File: app/config/config.yml (hunk 2 above)
Comment (HeahDude) [inline app/config/config.yml]: Nope.

**Comment by hhamon** [comment]
Need squash before merge.

**Pull request merged**